web/admin/src/Admin.js

```diff
@@ -129,8 +129,9 @@ export default class Admin extends PureComponent {
 
   onSessionNameChange = e => {
     const currentTitle = e.target.value.trim()
-    this.publicSessionRef().child(this.state.sessionId).update({name: e.target.value})
+    const isPublicSession = Object.keys(this.state.publicSessions).find(item => item === this.state.sessionId)
```

Comment:
This seems like a convoluted way of saying `!!this.state.publicSessions[this.state.sessionId]`

Reply:
Great point